src/main/scala/firrtl/Emitter.scala

```diff
@@ -90,14 +90,13 @@ class VerilogEmitter extends Emitter {
       case ClockType => e
       case AnalogType(w) => e
     }
-    (x) match {
+    x match {
```

Comment:
Do you have plans to convert these to extractor-style? (or, at least, the  ones that are accessing every field of the case class anyway like Mux, SubAccess)